code/obj/item/clothing/gloves.dm

```diff
@@ -68,6 +68,8 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 		if (length(newID))
 			return newID
 
+	proc/sheathe_blades()
+
```

Comment:
```suggestion
```
You don't need to do this, just define the proc on the `bladed` type and cast when you need it